.cspell/maintainer-words.txt

```diff
@@ -164,4 +164,9 @@ mohandas
 authy
 manavyalakinchara
 agam
-gitmodules
\ No newline at end of file
+gitmodules
+Boddu
+Kitiki
+BodduSriPavan-111
+Boddu Sri Pavan
+KitikiPlot
```

Comment:
Kitiki and KitikiPlot belong in the project-words list and not the maintainer-words list but that's okay, we can address that in a later PR.